tests/tests_integration/test_api/test_iam.py

```diff
@@ -1,82 +0,0 @@
-import pytest
```

Comment:
Why are we deleting these?

Reply:
See this thread: https://cognitedata.slack.com/archives/CAYSVGPUN/p1594724150121500